package.json

```diff
@@ -1089,6 +1090,11 @@
     "xvfb-maybe": "^0.2.1",
     "yargs-parser": "^20.2.9"
   },
+  "overrides": {
+    "mongodb-connection-model": {
+      "@mongodb-js/compass-utils": "0.3.4"
```

Comment:
`@mongodb-js/compass-utils` doesn’t export `getStoragePaths()` in the most recent version anymore which makes `require()`ing `mongodb-connection-model` fail, technically a breaking change and maybe we should have actually labelled https://github.com/mongodb-js/compass/commit/2c91060412b423097939120dd759492f497d31cb as `feat!:` rather than `chore(fs):` …

Reply:
I didn't know that vscode is so coupled with compass. Actually I didnt know that we are using legacy compass connections in vscode at all.